src/brouwers/shop/views.py

```diff
@@ -59,3 +59,9 @@ def post(self, request, *args, **kwargs):
         self.object = self.get_object()
         context = self.get_context_data(form=form, **kwargs)
         return self.render_to_response(context)
+
+
+class CartDetailView(DetailView):
+    queryset = Cart.objects.all()
```

Comment:
definitely needs a `def get_queryset` override that limits it to your own carts!

Especially with IDs in the URLs, this makes it very susceptible to an enumeration attack.